src/routes/dashboard/components/ChannelHealth.svelte

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
+  import { Column, Grid, Row, Tile } from 'carbon-components-svelte'
+  import { getChannelHealth } from '../helpers/channels'
+  import type { Questions } from '../types'
+
+  export let filteredQuestions: Map<string, Questions>
+
+  let data = getChannelHealth(filteredQuestions.get('aggregate')!)
+  console.log(data)
+
+  $: data = getChannelHealth(filteredQuestions.get('aggregate')!)
+</script>
+
+<svelte:head>
+  <title>Discord Metrics Dashboard</title>
+</svelte:head>
+
+<Grid class="styled-col" style="justify-content:center">
+  <h1 class="number-text">Channel Health</h1>
+  <div style="max-height:100vh; overflow-y:scroll">
```

Comment:
this can be targeted as `div` in the page's `style` block and Svelte will create a unique classname for it. Since this is the only `div` here we will be okay.

For what it's worth I typically avoid inline CSS unless absolutely necessary